config/azkaban/step.sh

```diff
@@ -36,5 +36,6 @@ sudo amazon-cloudwatch-agent-ctl -a append-config -m ec2 -c file:$CONFIG_FILE -s
 sudo chmod g+x /opt/emr/repos/chs-uc-lab/build_dataset.sh
 sudo chmod g+x /opt/emr/repos/chs-uc-lab/build_views.sh
 sudo chmod g+x /opt/emr/repos/chs-payment-timelines/scripts/build_pt.sh
```

Comment:
I'm beginning to think we just want a find -name '*.sh' -exec ...   to make .sh files executable!

Reply:
In fact can we remove all this sudo chmod g+x stuff and instead at the end of /home/hadoop/get_scripts.sh have that script find and set the execute flag on files ending in .sh

Reply:
removed the multiple chmods and added to get_scripts.sh - https://github.com/dwp/aws-analytical-env/pull/324